### DISCUSSION TITLE
Speed up starting of Martin

### DISCUSSION THREAD

**Discussion by IncidGeo** [maplibre/martin#1177] [Q&A] [answered]
Hi,
Every time i restart Martin it takes quite a long time before the message "Martin has been started" appear (15mn approximately)
- All my source have spatial index
- Some are quite large (87 millions of entries)

Custom configuration file
- Limit to 2 schemas but the database contain nearly 25
- All other settings to default

Should i define the bounding box for each source manually or is there any way to speed up the process ?

**Comment by sharkAndshark** [comment]
Maybe try set auto_bounds: skip

**Reply by IncidGeo** [reply]
I've tried 
auto_bounds : skip 
it's not faster.

Here is my config.yaml file :
postgres:
  connection_string: $DATABASE_URL
  auto_bounds: skip 
  auto_publish:
    from_schemas:
      - webapp
      - public

**Comment by IncidGeo** [comment]
I've activate the debug mode but i don't see anything wrong. This morning it took nearly an half hour to start and i need add much more table in the config file. Any guidance is welcome.

**Reply by nyurik** [reply]
In the debug mode, can you list all the queries Martin makes?  Or send me the log file

**Reply by nyurik** [reply]
also, after it runs for a few minutes (but before you kill it), open another PG connection in psql or some other tool, and query which queries are running. See https://stackoverflow.com/questions/12641676/how-to-get-a-status-of-a-running-query-in-postgresql-database

**Reply by nyurik** [reply]
OK, I think the issue was due to using old Martin that did not support bound check skipping properly.

**Comment by nyurik** [comment]
I added more debug-level logging in https://github.com/maplibre/martin/pull/1180 - once released (hopefully soon), it will be easier to trace the source of the bound-calc issues

**Reply by IncidGeo** [reply]
@nyurik,

I've send you an e-mail with all the logs on the mail you provide in github, let me know if you need more information.

**Reply by IncidGeo** [reply, marked as answer]
I've just upgrade the docker image to  v0.13.0, nearly instant start-up !!!!
